azurelinuxagent/common/protocol/extensions_goal_state.py

```diff
@@ -115,6 +119,7 @@ def compare_attribute(attribute):
             compare_attribute("status_upload_blob_type")
         compare_attribute("required_features")
         compare_attribute("on_hold")
+        compare_attribute("agent_manifests")
```

Comment:
Is the comparison of ext-handlers going to be done in a separate PR?

Reply:
yes, all this refactoring is affecting a lot of code and those changes need to be done gradually